### PR TITLE
DTGB-421: Add feedback form theming + some refactoring.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: gent_base.theme [hunk 1]
```diff
@@ -512,3 +512,16 @@ function _get_image_style_gallery_full(&$variables) {
     }
   }
 }
+
+/**
+ * Implements hook_theme_suggestions_alter().
+ */
+function gent_base_theme_suggestions_alter(array &$suggestions, array $variables) {
+  if (isset($variables['element']) && isset($variables['element']['#type']) && $variables['element']['#type'] == 'form') {
```
Comment (Jeroen005) [inline gent_base.theme]: Die eerste isset kan weg hier :)

File: styleguide/components/41-organisms/feedback-form/_feedback-form.scss [hunk 2]
```diff
@@ -18,24 +18,11 @@
...
-      width: span(3 of 9);
-      max-width: 7.8rem;
-      margin-right: 3.6rem;
-    }
-
-    &-text {
-      width: span(6 of 9);
+  form {
+    label {
+      display: none;
```
Comment (Jeroen005) [inline styleguide/components/41-organisms/feedback-form/_feedback-form.scss:23]: Is dit wel ok @delrueba ?

File: templates/custom/nodes/node--detail--layout.html.twig [hunk 3]
```diff
@@ -80,14 +80,22 @@
...
+  {{ title_suffix }}
+
+  {% block article_content_top %}
+    {{ content.field_summary }}
+  {% endblock %}
+
+  {% block article_content_summary %}
+  {% endblock %}
+
+  {% block article_content_bottom%}
```
Comment (Jeroen005) [inline templates/custom/nodes/node--detail--layout.html.twig]: spatie toevoegen achteraan

**Review by Jeroen005 — CHANGES_REQUESTED** [review]

File: styleguide/components/41-organisms/feedback-form/_feedback-form.scss (hunk 2 above)
Comment (meirege) [reply styleguide/components/41-organisms/feedback-form/_feedback-form.scss:23]: We hebben die discussie al gehad. In deze uitzondering is de titel genoeg en is het label niet meer nodig. Anders staat content dubbel.

File: gent_base.theme (hunk 1 above)
Comment (meirege) [reply gent_base.theme]: Fixed.

File: templates/custom/nodes/node--detail--layout.html.twig (hunk 3 above)
Comment (meirege) [reply templates/custom/nodes/node--detail--layout.html.twig]: Fixed.

**Review by Jeroen005 — APPROVED** [review]

**Pull request merged**